bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -55,12 +55,20 @@
 		<label>Current Electricity Price</label>
 		<description>Current price to import power from the grid</description>
 		<category>Price</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
I wonder if Forecast / Energy would be a better pair. The energy price shown is an estimate for the current 30min period/